cfme/tests/automate/test_git_import.py

```diff
@@ -78,3 +78,41 @@ def test_automate_git_domain_displayed_in_service(appliance, imported_domain, br
     view.modal.apply.click()
     assert view.provisioning_entry_point.value == ("/{}/Service/Provisioning/StateMachines/"
         "ServiceProvision_Template/CatalogItemInitialization".format(imported_domain.name))
+
+
+@pytest.mark.tier(3)
+@pytest.mark.parametrize(
+    ("url", "param_type", "param_value", "verify_ssl"),
```

Comment:
Question: Why are you parametrizing this test against a single parameter? I think this would better suited to a fixture, unless you plan on adding more parameters in the future.

Reply:
@john-dupuy 
This test case will not need more than existing parameters. Also I am actually dealing with only one git import url. As per my understanding, It would be better to have fixture; if I need to deal with multiple different urls with different parameters which  is not required in this test case. Hence keeping as it is.

Reply:
I agree with @john-dupuy  here. Why use parametrization if there is only one group of parameters? We use parameters because we need to test a single test case with multiple scenarios, and I only see one scenario here. You could simply define those variables inside the test case, I don't think there will be any major impact on test case if you do so.